package.json

```diff
@@ -254,7 +262,10 @@
     "ts-mixer@npm:^6.0.3": "patch:ts-mixer@npm%3A6.0.4#~/.yarn/patches/ts-mixer-npm-6.0.4-5d9747bdf5.patch",
     "sucrase@npm:3.34.0": "^3.35.0",
     "@expo/config/glob": "^10.3.10",
-    "@expo/config-plugins/glob": "^10.3.10"
+    "@expo/config-plugins/glob": "^10.3.10",
+    "@metamask/snaps-controllers@npm:^4.1.0": "patch:@metamask/snaps-controllers@npm%3A8.2.0#~/.yarn/patches/@metamask-snaps-controllers-npm-8.2.0-288806dfe2.patch",
+    "@metamask/snaps-controllers@npm:^3.4.1": "patch:@metamask/snaps-controllers@npm%3A8.2.0#~/.yarn/patches/@metamask-snaps-controllers-npm-8.2.0-288806dfe2.patch",
+    "@metamask/snaps-controllers@npm:^7.0.1": "patch:@metamask/snaps-controllers@npm%3A8.2.0#~/.yarn/patches/@metamask-snaps-controllers-npm-8.2.0-288806dfe2.patch"
```

Comment:
`snaps-controllers`'s ESM package is broken so we patch it so ESM can't be used.